doc/user_doc/vic_app_dev/using_volumes_with_vic.md

```diff
@@ -0,0 +1,66 @@
+# Using Volumes with vSphere Integrated Containers #
+
+vSphere Integrated Containers supports the use of container volumes. When you create or the vSphere Administrator creates a virtual container host, you or the Administrator specify the datastore to use to store container volumes in the `vic-machine create --volume-store` option. For information about how to use the `vic-machine create --volume-store` option, see the section on `volume-store` in [Virtual Container Host Deployment Options](../vic_installation/vch_installer_options.html#volume-store) in *vSphere Integrated Containers Installation and Configuration*.   
+
+## Obtain the List of Available Volume Stores ##
+
+To obtain the list of volume stores that are available on a virtual container host, run `docker info`.
+
+<pre>docker -H <i>virtual_container_host_address</i>:2376 --tls info</pre>
+
+The list of available volume stores for this virtual container host appears in the `docker info` output under `VolumeStores`.
+
+<pre>[...]
+Storage Driver: vSphere Integrated Containers Backend Engine
+VolumeStores: <i>volume_store_1</i> <i>volume_store_2</i> ... <i>volume_store_n</i>
+vSphere Integrated Containers Backend Engine: RUNNING
+[...]</pre>
+
+## Create a Volume in a Volume Store ##
+
+When you use the `docker volume create` command to create a volume, you can optionally provide a name for the volume by specifying the `--name` option. If you do not specify `--name`, `docker volume create` assigns a random UUID to the volume.
+
+If the volume store label is anything other than `default`, you must specify the `--opt VolumeStore` option and pass the name of an existing volume store to it. If you do not specify `--opt VolumeStore`, `docker volume create` searches for a volume store named `default`, and returns an error if no such volume store exists. 
```

Comment:
Added the following in [Create a Volume in a Volume Store](https://github.com/stuclem/vic/blob/master/doc/user_doc/vic_app_dev/using_volumes_with_vic.md#create-a-volume-in-a-volume-store) (for container app developers):

If you intend to create anonymous volumes by using `docker create -v`, a volume store named `default` must exist.

**NOTE**: If you use `docker create -v`, vSphere Integrated Containers only supports the `-r` and `-rw` options.

<pre>docker -H <i>virtual_container_host_address</i>:2376 --tls create 
-v <i>mount_path</i></pre>

Added the following in the description of the `vic-create` [volume-store option](https://github.com/stuclem/vic/blob/master/doc/user_doc/vic_installation/vch_installer_options.md#volume-store) (for vSphere admins/whoever runs `vic-machine create`):

  **NOTE**: If container application developers intend to create anonymous volumes by using `docker create -v`, you must create a volume store with a label of `default`.